internal/pkg/tekton/pipeline_run_builder.go

```diff
@@ -211,6 +232,43 @@ func NewPipelineRunBuilder(name, namespace string) *PipelineRunBuilder {
 												},
 											},
 										},
+										{ // Run even if step-renovate fails
+											Name:   "log-analyzer",
+											Image:  "quay.io/konflux-ci/renovate-log-analyzer:latest",
+											Script: "",
+											SecurityContext: &corev1.SecurityContext{
+												Capabilities:             &corev1.Capabilities{Drop: []corev1.Capability{"ALL"}},
+												RunAsNonRoot:             ptr.To(true),
+												RunAsUser:                &normalUser,
+												AllowPrivilegeEscalation: ptr.To(false),
+											},
+											Env: []corev1.EnvVar{
+												{
+													Name: "POD_NAME",
+													ValueFrom: &corev1.EnvVarSource{
+														FieldRef: &corev1.ObjectFieldSelector{
+															FieldPath: "metadata.name",
+														},
+													},
+												},
+												{
+													Name: "NAMESPACE",
+													ValueFrom: &corev1.EnvVarSource{
+														FieldRef: &corev1.ObjectFieldSelector{
+															FieldPath: "metadata.namespace",
```

Comment:
Which namespace should be specified in the Kite requests? Is it always "mintmaker", or should it be the namespace of the component?
If it's the latter, then you should use the `mintmaker.appstudio.redhat.com/namespace` label instead.